index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   admission-controller:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2021-10-19T13:26:13.082176-07:00"
+    created: "2021-11-16T19:49:36.098775-08:00"
```

Comment:
We have to revert this time. Created time for older helm charts should not change. The --merge option should take care of it but it doesn't. Hence it's a manual step.